boards/arm/stm32h7/nucleo-h745zi/src/stm32_bringup.c

```diff
@@ -227,3 +231,13 @@ int stm32_bringup(void)
 
   return OK;
 }
+
+#ifdef CONFIG_SYSTEMTICK_HOOK
+
+sem_t g_waitsem;
```

Comment:
where the semaphore is consumed?

Reply:
Semaphore will be consumed in pysimCoder 

File:    nuttx_main_timehook.c

```
[...]
extern sem_t g_waitsem;

[...]
  while(!end){
    while (sem_wait(&g_waitsem) >= 0) {
	if(nTick==0){

[...]
....